extensions/amp-analytics/0.1/variables.js

```diff
@@ -284,17 +284,17 @@ export class VariableService {
   }
 
   /**
+   * @param {string} unusedName Name of the variable. Only used in tests.
    * @param {string|!Array<string>} raw The values to URI encode.
-   * @param {string} unusedName Name of the variable.
    * @return {string} The encoded value.
    */
-  encodeVars(raw, unusedName) {
+  encodeVars(unusedName, raw) {
```

Comment:
It looks like `unusedName` is actually being used in [amp-analytics.js:245](https://github.com/ampproject/amphtml/pull/12459/files#diff-b951eee683ce1cc97c5e22f9abd32f11R245 ), so it should be called `name` instead of `unusedName`

Reply:
did you mean requests.js:245? yes the caller passed in the `name`, but it's not used in this function.

Reply:
Hmm ok I think I understand now.